### ISSUE TITLE
Publish grestful on DUB

### ISSUE THREAD

**Issue by azbukagh** [Gert-dev/grestful#15]
More developers will get involved and grestful can be installed via `dub fetch grestful` and used via `dub run grestful`.

Why grestful not on http://code.dlang.org yet?

**Comment by Gert-dev** [comment]
No explicit reason. I figured that code.dlang.org was mostly useful for libraries as I don't think anyone will be looking to install grestful as a dependency for another project.

**Comment by azbukagh** [comment]
Yup, there is no reason to use grestful as a dependency, but code.dlang.org is a great way to install and use applications. You don't have to pay or something to publish package on DUB. It costs nothing, but it can involve a lot of developers.

grestful has only one problem: it's almost dead. Only one way to solve this problem is spread the word about it.
